src/renderers/shared/utils/Transaction.js

```diff
@@ -229,6 +236,8 @@ var Mixin = {
   },
 };
 
+export type TransactionType = typeof Transaction.Mixin;
```

Comment:
also, the way I wrote it works by accident, Transaction.Mixin doesn't exist yet in this line :p